docs/concept/persistence-storage.md

```diff
@@ -100,7 +100,7 @@ return: File pointer, you can use it for various operations.
 
 ### Write
 BanyanDB provides two methods for writing files.
-Append mode, which adds new data to the end of a file. This mode is typically used for WAL.
+Append mode, which adds new data to the end of a file. This mode is typically used for WAL. And BanyanDB supports vector Append mode, which supports appending consecutive buffers to the end of the file.
```

Comment:
All append operations are "vector" style. Can you elaborate why we should highlight it.

Reply:
A 'vector' refers to a continuous set of memory buffers. By using the writev system call, multiple buffers (i.e. vectors) can be written in one operation.
In writev, it is necessary to provide a vector composed of iovec structures, each describing the address and length of a buffer to be written. The content of this vector is equivalent to an array, hence it is called a "write vector".

Reply:
That is what linux define.